File: 태성/0611/약수의개수.py
left = 13
right = 17

ans = 0
for a in range(left, right+1):
    cnt = 0
    for b in range(1, a+1):
        if a % b == 0:
            cnt +=1

    if cnt % 2 == 0:
        ans += a

    else:
        ans -= a

print(ans)

File: 태성/LV1/0611/약수의개수.js
function solution(left, right) {
    var answer = 0
    for (i=left; i<right+1; i++) {
        var cnt = 0
        for (a=1; a<i+1; a++) {
            if (i % a === 0) {
                cnt +=1
            }
        }

        if (cnt % 2 === 0) {
            answer += i
        } else {
            answer -= i
        }
    }

    return answer
}

console.log(solution(13, 17))